Description: get the correct multi-arch triplet
 On certain architectures, epoll is not correctly picked up because of
 incorrect multi-arch triplet.
Author: Vasudev Kamath <kamathvasudev@gmail.com>
Forwarded: no
Reviewed-By: Ramakrishnan Muthukrishnan <rkrishnan@debian.org>
Last-Update: 2016-01-20

--- a/mk/re.mk
+++ b/mk/re.mk
@@ -197,7 +197,7 @@
 # OS section
 #
 
-MACHINE   := $(shell $(CC) -dumpmachine)
+MACHINE   := $(shell $(CC) --print-multiarch)
 
 ifeq ($(CROSS_COMPILE),)
 OS        := $(shell uname -s | sed -e s/SunOS/solaris/ | tr "[A-Z]" "[a-z]")
